src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -291,6 +312,12 @@ public void setBases( final byte[] bases ) {
         return convertedBaseQualities;
     }
 
+    @Override
+    public byte[] getBaseQualitiesNoCopy() {
+        // Not possible to have a no-copy implementation for Google reads
+        return getBaseQualities();
```

Comment:
these should either throw as not implemented or the description of the semantics needs to change in GATKRead

Reply:
Description in `GATKRead` changed